cluster/gce/trusty/node.yaml

```diff
@@ -162,8 +163,16 @@ start on stopped kube-env
 
 script
 	. /etc/kube-env
-	# If kubelet or kube-proxy is not installed in the image, pull release binaries and put them in /usr/bin.
-	if ! which kubelet > /dev/null || ! which kube-proxy > /dev/null; then
+	# If both kubelet and kube-proxy test args are set, this is a testing cluster.
+	# We pull binaries and put them in /usr/local/bin. If it is not a testing
```

Comment:
So I guess we are assuming that non-test clusters that don't have existing binaries have a writeable /usr/bin directory.

I think it might be clearer to just have a variable to force install the binaries (to /usr/local/bin) and set it for testing. 

The script to run the binary would look in /usr/local/bin first and fall back to /usr/bin if the binary doesn't exist. If it's in neither place (because it wasn't preinstalled and you didn't specify that you wanted it) then the node would just fail to start the binary.